app/views/applications/show.html.erb

```diff
@@ -0,0 +1,80 @@
+<h1>Applicant's Name: <%= @application.name%></h1>
+<p> Street Address: <%= @application.street_address%></p>
+<p>City: <%= @application.city%></p>
+<p>State: <%= @application.state%></p>
+<p>Zip Code: <%= @application.zip_code%></p>
+<p>Description: <%= @application.description%></p>
+<p>Status: <%= @application.status%></p>
+
+
+
+<section class="adoptpet">
+
+  <h1>Application Pets</h1>
+    
+  <% @application.pets.each do |pet| %>
+    <h2><%= pet.name%></h2> 
+    
+  <% end %>
+
+</section>
+
+<% @application.pets.each do|pet|%>
+<%= link_to "#{pet.name}", "/pets/#{pet.id}" %>
+<%end%>
+
+
+
+
+<section class="pet">
+  <%= form_with url: "/applications/#{@application.id}", method: :get  do |f| %>
+    <div>
+      <%= f.label :pet_name%>
+      <%= f.text_field :pet_name %>
+    </div>
+    <div>
+      <%= f.submit 'Search' %>
+
+
+    </div>
+  <% end %>
+    <section class="us8">
+      
+    </section>
+  <% if @pets != nil  %>  
+    <% @pets.each do |pet| %>
+      
+      <h2><%= pet.name%></h2>
+      
+
+      <%= form_with(url:"/applications/#{@application.id}/pets", method: :post ) do |f| %>
+        <div>
+          <%= f.hidden_field :pet_id, value: pet.id %>
+          <%= f.submit 'Adopt this pet' %>
+        </div>
+      <% end %>
+
+    <% end %>
+  <% end %>
+</section>
+
+<section class="why_I_would_make_a_good_owner">
+  <section class="nopets">
+ 
+    <% if @application.pets.present? %>
+      <% if @application.status == "In Progress" %>
+          <%= form_with(url: "/applications/#{@application.id}?why_i_would_make_a_good_owner=true", method: :patch ) do |f| %>
```

Comment:
Would this functionality still work if you took out `why_i_would_make_a_good_owner=true` from the path? You're checking for the presence of that param in the controller, so just having a user fill out the form should add that parameter, and you shouldn't need it in the path as a query parameter as well